pkg/gator/verify/read_suites.go

```diff
@@ -100,11 +100,14 @@ func readSuites(f fs.FS, files []string, originalPath string) ([]*Suite, error)
 		if suite != nil {
 			suite.AbsolutePath = file
 
-			// trim any prefixes like "/" or "./" in order for the
+			// trim any prefixes like "/", "./" or "../" in order for the
 			// .Cut call below to actually work with the absolute path
 			// contained in the file var.
 			cutPath := strings.TrimPrefix(originalPath, "/")
 			cutPath = strings.TrimPrefix(cutPath, "./")
+			for strings.HasPrefix(cutPath, "../") {
```

Comment:
i'm not too concerned about an infinite loop here since specifying an arbitrarily large number of prev directories has to happen for that